graphql_jwt/decorators.py

```diff
@@ -97,3 +98,33 @@ def on_resolve(values):
             return Promise.resolve(values).then(on_resolve)
         return on_resolve(values)
     return wrapper
+
+
+def setup_jwt_cookie(f):
+    @wraps(f)
+    def wrapper(cls, root, info, *args, **kwargs):
+        result = f(cls, root, info, **kwargs)
+
+        if getattr(info.context, 'jwt_cookie', False):
+            info.context.jwt = result.token
+        return result
+    return wrapper
+
+
+def jwt_cookie(view_func):
+    @wraps(view_func)
+    def wrapped_view(request, *args, **kwargs):
+        request.jwt_cookie = True
+        response = view_func(request, *args, **kwargs)
+
+        if hasattr(request, 'jwt'):
+            expiration = datetime.utcnow() + jwt_settings.JWT_EXPIRATION_DELTA
+
+            response.set_cookie(
```

Comment:
What do you think about making the cookie secure only as an opt-in? Similar to https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-SESSION_COOKIE_SECURE

Reply:
Good idea, thanks!